packages/lit/src/components/wizard-token-input/index.ts

```diff
@@ -67,6 +72,11 @@ export class WizardTokenInput extends LitElement {
 
   override render() {
     return html` <label for=${this.id}>${this.label}</label>
+      ${this.errors.length
+        ? html`<ul class="error">
+            ${this.errors.map(({ issue }) => html`<li>${issue.message}</li>`)}
+          </ul>`
+        : nothing}
```

Comment:
```suggestion
      ${this.errors.length
        && html`<ul class="error">
            ${this.errors.map(({ issue }) => html`<li>${issue.message}</li>`)}
          </ul>`}
```

Scheelt weer een nothing import.